src/tracing.rs

```diff
@@ -1,8 +1,12 @@
+use std::io::IsTerminal as _;
 use tracing_subscriber::{prelude::*, EnvFilter};
 
 /// Initializes a tracing subscriber.
 pub(crate) fn init_subscriber() -> anyhow::Result<()> {
-    let fmt_layer = tracing_subscriber::fmt::layer().with_target(true);
+    let fmt_layer = tracing_subscriber::fmt::layer()
+        .with_target(true)
+        .with_ansi(std::io::stderr().is_terminal())
```

Comment:
logging to stderr feels _mildly_ surprising to me, but i don't have a strong opinion about this 👍

Reply:
Yeah, subjective, I guess. Putting on my sysadmin hat, I feel strongly that "stdout is for output, stderr is for logging and diagnostics." But in the case of `galileo serve`, where the _only_ output is logging, stdout is probably fine. Old habits die hard.